pyuploadcare/api_resources.py

```diff
@@ -711,6 +713,60 @@ class FileList(BaseApiList):
     filters = [('stored', None), ('removed', False)]
 
 
+class FilesStorage(object):
+    """ Object for doing storage-related operations like ``store``, ``delete``
```

Comment:
Batch storage operations for the list of files. ``store`` and ``delete`` are supported.